static/js/dark.js

```diff
@@ -0,0 +1,53 @@
+// JavaScript (assets/js/darkmode.js)
+var themeLink = document.getElementById("theme-link");
+var darkModeFile = "/css/dark.css" ;
+var lightModeFile = "/css/global.css";
```

Comment:
absolute path may cause some problem if user set the `baseUrl` with subpath

Reply:
i did try doing that but it didn't work well.